odl/operator/operator.py

```diff
@@ -1158,8 +1158,7 @@ def adjoint(self):
 
     def __repr__(self):
         """Return ``repr(self)``."""
-        return '{}({!r}, {!r})'.format(self.__class__.__name__,
-                                       self.left, self.right)
+        return '({!r} + {!r})'.format(self.left, self.right)
```

Comment:
Not sure I like this change. For long and messy output (which is like 90 % of the time), there's no way to use multiple lines for better visual appearance. Also the "+" sign will be hard to find in the output, whereas `OperatorSum` is clear from the beginning.

For `__str__`, okay.